integration_test/scripts/publish-images.sh

```diff
@@ -1,14 +1,37 @@
 #!/bin/bash
-# imageProject publishFlag
+# imageProject publishFlag repositoryString
+
+accessToken=$ACCESS_TOKEN
+repositoryString=$3
+IFS='_' read -r -a repoArray <<< "$repositoryString"
+githubUser="${repoArray[1]}"
+githubRepo="${repoArray[2]}"
+if [ -z "$githubUser" ]
+then
+    githubUser="GoogleCloudPlatform"
+    githubRepo=$3
+fi
 
 base_name="gcr.io/${1}/healthcare-api-dicom-dicomweb-adapter-"
 
 publish_adapter () {
   adapter_name=${base_name}${1}
-  docker push $adapter_name
+  docker push $adapter_name > /dev/null 2> /dev/null
+  echo "https://${adapter_name}:${2}"
 }
 if [ "$2" == "true" ]
 then
-  publish_adapter import
-  publish_adapter export
+  version=`docker images | grep $base_name | awk '{print $2}' | grep "\." | head -n 1`
```

Comment:
Can you add a comment explaining what the steps do here? It's a bit hard to understand from just the command alone.

Reply:
done

Reply:
For my understanding what is the format of the output passed to grep ".\"?

Reply:
> For my understanding what is the format of the output passed to grep "."?

strings "x.y.z" and hashcode, by `grep "\."` we remove hashcode

Reply:
thanks, could you add that to the comment?

Reply:
done